backend/src/graphql_api/stable_coin.rs

```diff
@@ -425,4 +434,57 @@ impl QueryStableCoins {
         );
         txn_summary
     }
+
+    async fn live_stablecoins(&self, _ctx: &Context<'_>) -> anyhow::Result<Vec<StableCoin>> {
+        let endpoint_env = env::var("CCDSCAN_INDEXER_GRPC_ENDPOINTS");
+        if endpoint_env.is_err() {
+            return Err(anyhow::anyhow!(
+                "Stablecoin data is not available. Only available in Devnet"
+            ));
+        }
+        let app = App::parse();
+        let mut client = v2::Client::new(app.endpoint).await.context("Failed to create client")?;
```

Comment:
No need to do the `App::parse()` here, which is also only designed to be used from the main thread
```suggestion
        let endpoint: v2::Endpoint = endpoint_env.context("Stablecoin data is not available. Only available in Devnet")?.parse().context("Failed parsing the node endpoint from env CCDSCAN_API_GRPC_ENDPOINT")?;
        let mut client = v2::Client::new(endpoint).await.context("Failed to create client")?;
```